### DISCUSSION TITLE
Test Coverage Error

### DISCUSSION THREAD

**Discussion by Stevepurpose** [vitest-dev/vitest#4691] [Q&A]
Please why do I get this error message when I try to add coverage to my previously passing tests. Also from the coverage report, they are at 0%

Error: Invalid transform mode: undefined
 ❯ V8CoverageProvider.onAfterSuiteRun node_modules/@vitest/coverage-v8/dist/provider.js:232:13
 ❯ Proxy.onAfterSuiteRun node_modules/vitest/dist/vendor-node.a7c48fe1.js:7143:57
 ❯ MessagePort.<anonymous> node_modules/vitest/dist/vendor-index.b271ebe4.js:59:29
 ❯ [nodejs.internal.kHybridDispatch] node:internal/event_target:757:20
 ❯ exports.emitMessage node:internal/per_context/messageport:23:28

This error originated in "add.test.js" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.

**Comment by AriPerkkio** [comment]
This error can only come up if you are mixing old version of `vitest` package and new version of `@vitest/coverage-*` package. 
Update all of your Vitest packages to latest. 

Based on the stack trace you are using `vitest@0.34.6`: 
> ❯ Proxy.onAfterSuiteRun node_modules/vitest/dist/vendor-node.a7c48fe1.js:7143:57
> https://unpkg.com/browse/vitest@0.34.6/dist/

**Comment by Stevepurpose** [comment]
Thanks, but please what command can I use for these updates.

**Reply by AriPerkkio** [reply]
`npm install --save-dev vitest@latest` if you are using `npm`.

**Comment by Stevepurpose** [comment]
Ok thanks a lot.

**Comment by Stevepurpose** [comment]
Do I need to uninstall the version I have first.

**Reply by AriPerkkio** [reply]
It's not required but should work both ways.

**Comment by Stevepurpose** [comment]
Ok thanks, it's working now.
